# A custom message type for sending commands to a Franka gripper
# Define the message fields
bool isStop